cypress/e2e/CMS/$InfographicCreation.feature

```diff
@@ -0,0 +1,101 @@
+        Then user is logged in and the user name "admin" is displayed in the toolbar
+        And the tool bar appears at the top
+        When user clicks on "Content" tab
+        And user clicks on "Media" sub tab
+        And user clicks on "Add media" action button
+        And user clicks on "Infographic" content type
+        Then page title is "Add Infographic"
+        When user selects test site section
+        And user fills out the following fields
+            | fieldLabel               | value                                   | field_name                     |
+            | Pretty URL               | test-infographic                        | field_pretty_url               |
+            | Name                     | Test Infographic                        | name[0][value]                 |
+            | Browser Title            | Test Infographic                        | field_browser_title            |
+            | Meta Description         | Automated Test Infographic              | field_page_description         |
+            | Card Title               | Automated Test Infographic - Card Title | field_card_title               |
+            | Feature Card Description | Test Infographic Card Description       | field_feature_card_description |
+        And user types "Test Caption" into Caption text field
+        And user uploads infographic "infographic-desktop.jpeg." as Desktop Version
+        And system waits for file upload process
+        And user uploads infographic "infographic-mobile.jpeg." as Mobile Version
+        And system waits for file upload process
+        And user selects "Published" from "Save as" dropdown
+        Then user saves the content page
+
+
+    Scenario: Create test pages to test infographic
+        Given user is navigating to "/user/login"
+        When user enters credentials
+        And user clicks "Log in" button
+        Then user is logged in and the user name "admin" is displayed in the toolbar
+        And the tool bar appears at the top
+        When user clicks on "Content" tab
+        And user clicks on "Add content" action button
+        And user clicks on "Article" content type
+        Then page title is "Create Article"
+        When user selects test site section
+        And user fills out the following fields
+            | fieldLabel       | value                                          | field_name             |
+            | Pretty URL       | article-to-test-infographic                    | field_pretty_url       |
+            | Page Title       | Article to test infographic                    | title[0][value]        |
+            | Browser Title    | Article to test infographic                    | field_browser_title    |
+            | Meta Description | Article to test infographic - Meta Description | field_page_description |
+        ***STEPS TO EMBED INFOGRAPHIC***
+        And user selects "Published" from "Save as" dropdown
```

Comment:
I suggest to not publish it until you embed infographic ( so just save as draft)